grails-app/assets/javascripts/customer/CustomerShowController.js

```diff
@@ -0,0 +1,87 @@
+function CustomerShowController() {
+    this.reference = document.querySelector(".background-box")
+    var _this = this;
+    var postalCodeValidator
+
+    this.init = function() {
+        postalCodeValidator = new PostalCodeValidator();
+        document.getElementById("postalCode").addEventListener("blur", _this.processPostalCodeInput);
```

Comment:
this.reference.find("postalCode") não funcionou aqui?

Reply:
eu testei e não funcionou, diz que não é uma função. Não sei se é algo do JQuery, se for talvez seja isso